src/parser/addressRepresentationConverters.ts

```diff
@@ -166,7 +178,7 @@ export function columnIndexToLabel(column: number) {
   return result.toUpperCase()
 }
 
-export function sheetIndexToString(sheetId: number, sheetMappingFn: SheetIndexMappingFn): string | undefined {
+export function sheetIndexToString(sheetId: number, sheetMappingFn: SheetIndexMappingFn): Maybe<string> {
```

Comment:
in some places you've changed `Maybe<string>` into `string | undefined` but here the opposite?

Reply:
I changed to `T | undefined` in the `HyperFormula.ts` so the API does not introduce extra object.
Internally I prefer we use `Maybe<T>` wherever it makes sense.

Reply:
Sounds legit 👍